Lib/fontTools/feaLib/parser.py

```diff
@@ -39,7 +39,7 @@ class Parser(object):
     CV_FEATURE_TAGS = {"cv%02d" % i for i in range(1, 99+1)}
 
     def __init__(self, featurefile, glyphNames=(), followIncludes=True,
-                 **kwargs):
+                 include_dir=None, **kwargs):
```

Comment:
the rest of the parameters use camelCase, so it should be `includeDir` here. I know..